utils/json_profiles_merger.py

```diff
@@ -18,6 +18,7 @@
 
 FLAGS = flags.FLAGS
 flags.DEFINE_string('output_path', None, 'The path to the output file.')
+flags.mark_flag_as_required("output_path")
```

Comment:
One final nit: please use single quote to be consistent with the rest of the file.
(should have mentioned this in my first review. Sorry about that.)